packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -692,3 +693,459 @@ class MenuOptionMultiSelect extends React.Component {
+                    Docker File
+                  </MenuItem>
+                </DrilldownMenu>
+              }
+            >
+              Add storage
+            </MenuItem>
+            <MenuItem itemId="edit">Edit</MenuItem>
+            <MenuItem itemId="delete_deployment">Delete deployment config</MenuItem>
+          </MenuList>
+        </MenuContent>
+      </Menu>
+    );
+  }
+}
+```
+
+### With drilldown breadcrumbs
+
+```js
+import React from 'react';
+import {
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem,
+  Divider,
+  DrilldownMenu,
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbHeading,
+  MenuBreadcrumb,
+  Dropdown,
+  DropdownToggle
+} from '@patternfly/react-core';
+import StorageDomainIcon from '@patternfly/react-icons/dist/js/icons/storage-domain-icon';
+import CodeBranchIcon from '@patternfly/react-icons/dist/js/icons/code-branch-icon';
+import LayerGroupIcon from '@patternfly/react-icons/dist/js/icons/layer-group-icon';
+import CubeIcon from '@patternfly/react-icons/dist/js/icons/cube-icon';
+
+class MenuWithDrilldown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      menuDrilledIn: [],
+      drilldownPath: [],
+      menuHeights: {},
+      activeMenu: 'rootMenu'
+    };
+    this.drillIn = (fromMenuId, toMenuId, pathId) => {
+      this.setState({
+        menuDrilledIn: [...this.state.menuDrilledIn, fromMenuId],
+        drilldownPath: [...this.state.drilldownPath, pathId],
+        activeMenu: toMenuId
+      });
+    };
+    this.drillOut = toMenuId => {
+      const indexOfMenuId = this.state.menuDrilledIn.indexOf(toMenuId);
+      const menuDrilledInSansLast = this.state.menuDrilledIn.slice(0, indexOfMenuId);
+      const indexOfMenuIdPath = this.state.drilldownPath.indexOf(toMenuId);
+      const pathSansLast = this.state.drilldownPath.slice(0, indexOfMenuIdPath);
+      this.setState({
+        menuDrilledIn: menuDrilledInSansLast,
+        drilldownPath: pathSansLast,
+        activeMenu: toMenuId
+      });
+    };
+    this.setHeight = (menuId, height) => {
+      if (!this.state.menuHeights[menuId]) {
+        this.setState({
+          menuHeights: {
+            ...this.state.menuHeights,
+            [menuId]: height
+          }
+        });
+      }
+    };
+  }
+
+  render() {
+    const { menuDrilledIn, drilldownPath, activeMenu, menuHeights } = this.state;
+    return (
+      <Menu
+        id="rootMenu"
+        containsDrilldown
+        drilldownItemPath={drilldownPath}
+        drilledInMenus={menuDrilledIn}
+        onDrillIn={this.drillIn}
+        onDrillOut={this.drillOut}
+        onGetMenuHeight={this.setHeight}
+      >
+        <MenuContent menuHeight={`${menuHeights[activeMenu]}px`}>
+          <MenuList>
+            <MenuItem
+              itemId="group:start_rollout"
+              direction="down"
+              drilldownMenu={
+                <DrilldownMenu id="drilldownMenuStart">
+                  <MenuBreadcrumb>
```

Comment:
In core this is constructed a bit differently. The breadcrumbs aren't a list item adjacent to the other items in the dropdown menu list. `.pf-c-menu__breadcrumb` (`<MenuBreadcrumb>`) is a direct child of `.pf-c-menu` (`<Menu>`). After that is a `<Divider>` (`<hr>`), then `.pf-c-menu__content` (`<MenuContent>`) comes after the divider. Here's a screenshot:

<img width="435" alt="Screen Shot 2021-03-02 at 5 47 53 PM" src="https://user-images.githubusercontent.com/35148959/109730773-a557f480-7b7f-11eb-9639-8d635932b6a7.png">

That said, we can update this if the markup needs to be different. I'm curious what @jessiehuff thinks. At a minimum though, if we keep the markup in this PR and update core to match, `<MenuBreadcrumb>` needs to be an `<li>` since it's a child of `<MenuList>` (a `<ul>`)

cc @mattnolting

Reply:
The back/up links follow the menus as you drillin and are relative to them, which I believe is the desired effect. 

`<MenuBreadcrumb>` is a child of `pf-c-menu`, rather than `<MenuList>`, so it should be a `<div>`. Otherwise you see a new set of breadcrumbs scroll in with each drilldown, which I don't think is the desired effect. @mcarrano @doruskova wdyt?